api/service/block.go

```diff
@@ -0,0 +1,108 @@
+package service
+
+import (
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/photon-storage/go-photon/sak/time/slots"
+	pbc "github.com/photon-storage/photon-proto/consensus"
+
+	"github.com/photon-storage/photon-explorer/api/pagination"
+	"github.com/photon-storage/photon-explorer/database/orm"
+)
+
+type latestBlock struct {
+	Epoch     uint64 `json:"epoch"`
+	Slot      uint64 `json:"slot"`
+	Position  uint64 `json:"position"`
+	Timestamp uint64 `json:"timestamp"`
+}
+
+// LatestBlocks handles the /latest-blocks request.
+func (s *Service) LatestBlocks(
+	_ *gin.Context,
+	page *pagination.Query,
+) (*pagination.Result, error) {
+	blks := make([]*orm.Block, 0)
+	if err := s.db.Model(&orm.Block{}).Offset(page.Start).
```

Comment:
Next time, when we write func calls in a chain, let's use the style similar to the mock. One call per line.